### PR TITLE
Ravinder | Test | Integrate Angular material on petty cash

### PR DESCRIPTION
**Pull request by ravinderwalkover**
https://app.clickup.com/t/6n147m

### CODE REVIEW

**Review by prateek-giddh — APPROVED** [review]

File: apps/web-giddh/src/app/expenses/components/approve-petty-cash-entry-confirm-dialog/approve-petty-cash-entry-confirm-dialog.component.html [hunk 1]
```diff
@@ -53,8 +53,16 @@ <h3 class="modal-title pull-left">{{ localeData?.approve }}</h3>
...
 
         <div>
-            <button class="btn btn-warning" (click)="approveEntry.emit(false)">{{ commonLocaleData?.app_no }}</button>
-            <button class="btn btn-success" (click)="approveEntry.emit(true)" [ladda]="approveEntryRequestInProcess">
+            <button mat-raised-button color="accent" (click)="approveEntry.emit(false)">
+                {{ commonLocaleData?.app_no }}
+            </button>
+            <button
+                mat-raised-button
+                color="primary"
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/expenses/components/approve-petty-cash-entry-confirm-dialog/approve-petty-cash-entry-confirm-dialog.component.html:61]: remove the color and use class name properly. @ravinderwalkover

File: apps/web-giddh/src/app/expenses/components/approve-petty-cash-entry-confirm-dialog/approve-petty-cash-entry-confirm-dialog.component.html [hunk 2]
```diff
@@ -53,8 +53,16 @@ <h3 class="modal-title pull-left">{{ localeData?.approve }}</h3>
         </div>
 
         <div>
-            <button class="btn btn-warning" (click)="approveEntry.emit(false)">{{ commonLocaleData?.app_no }}</button>
-            <button class="btn btn-success" (click)="approveEntry.emit(true)" [ladda]="approveEntryRequestInProcess">
+            <button mat-raised-button color="accent" (click)="approveEntry.emit(false)">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/expenses/components/approve-petty-cash-entry-confirm-dialog/approve-petty-cash-entry-confirm-dialog.component.html:56]: remove the color and use class name properly. @ravinderwalkover

File: apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html [hunk 3]
```diff
@@ -5,13 +5,24 @@
                 <p class="bold mr-b05">{{ selectedItem?.entryDate }} | {{ selectedItem.baseAccount.name }}</p>
             </div>
             <div class="col-sm-6 text-right xs-left">
-                <button type="button" class="mr-b05 btn btn-success" (click)="showApproveConfirmPopup(ApproveConfirm)">
+                <button
+                    mat-raised-button
+                    color="primary"
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html:10]: remove the color and use class name properly. @ravinderwalkover

File: apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html [hunk 4]
```diff
@@ -5,13 +5,24 @@
...
+                    (click)="showApproveConfirmPopup(approveConfirm)"
+                >
                     {{ localeData?.approve }}
                 </button>
-                <button type="button" class="mr-b05 btn btn-danger" (click)="openModal(RejectionReason)">
+                <button
+                    mat-raised-button
+                    color="accent"
+                    type="button"
+                    class="mr-l1"
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html:20]: remove the color and use class name properly. @ravinderwalkover

File: apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html [hunk 5]
```diff
@@ -172,26 +186,28 @@ <h3 class="modal-title pull-left">{{ localeData?.rejection_reason }}</h3>
...
+                    id="rejectReason"
+                    [formControl]="rejectReason"
+                    class="textarea form-control mat-field-border mt-2"
+                    [placeholder]="localeData?.reason_placeholder"
+                ></textarea>
+            </mat-form-field>
         </div>
 
-        <button type="submit" class="btn btn-success" [disabled]="!rejectReason.value" (click)="submitReject()">
+        <button mat-raised-button color="primary" [disabled]="!rejectReason.value" (click)="submitReject()">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html:202]: remove the color and use class name properly. @ravinderwalkover

File: apps/web-giddh/src/app/expenses/components/filter-list/filter-list.component.scss [hunk 6]
```diff
@@ -4,33 +4,32 @@
     width: 100%;
 }
 
-.filter-list .inner-filter-list li {
-    border-bottom: 1px solid $gainsboro;
-    padding: 15px;
-    display: flex;
+.filter-list .inner-filter-list .mat-list-item {
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/expenses/components/filter-list/filter-list.component.scss:7]: use scss for this, not css @ravinderwalkover

File: apps/web-giddh/src/app/expenses/components/pending-list/pending-list.component.scss [hunk 7]
```diff
@@ -62,7 +62,9 @@
 .td-search-box {
     position: relative;
 }
-
+.mat-column-s_no {
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/expenses/components/pending-list/pending-list.component.scss:65]: change this classname and add proper class, according to our guideline @ravinderwalkover

File: apps/web-giddh/src/app/expenses/components/rejected-list/rejected-list.component.scss [hunk 8]
```diff
@@ -49,3 +49,12 @@
 .icon-pointer {
     cursor: pointer;
 }
+.mat-column-s_no {
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/expenses/components/rejected-list/rejected-list.component.scss:52]: change this classname and add proper class, according to our guideline @ravinderwalkover

File: apps/web-giddh/src/app/expenses/components/approve-petty-cash-entry-confirm-dialog/approve-petty-cash-entry-confirm-dialog.component.html (hunk 1 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/expenses/components/approve-petty-cash-entry-confirm-dialog/approve-petty-cash-entry-confirm-dialog.component.html:61]: Done

File: apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html (hunk 3 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html:10]: Done

File: apps/web-giddh/src/app/expenses/components/approve-petty-cash-entry-confirm-dialog/approve-petty-cash-entry-confirm-dialog.component.html (hunk 2 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/expenses/components/approve-petty-cash-entry-confirm-dialog/approve-petty-cash-entry-confirm-dialog.component.html:56]: Done

File: apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html (hunk 4 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html:20]: Done

File: apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html (hunk 5 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html:202]: Done

File: apps/web-giddh/src/app/expenses/components/pending-list/pending-list.component.scss (hunk 7 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/expenses/components/pending-list/pending-list.component.scss:65]: @aditi-tiwari17 this is dynamic class, can't control it.

File: apps/web-giddh/src/app/expenses/components/rejected-list/rejected-list.component.scss (hunk 8 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/expenses/components/rejected-list/rejected-list.component.scss:52]: @aditi-tiwari17 this is dynamic class, can't control it.

**Pull request merged**